admin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bens Admin Page</title>
+    <link rel='stylesheet' type='text/css' href='css/normalise.css'>
+    <link rel='stylesheet' type='text/css' href='css/admin.css'>
+</head>
+
+<body>
+    <header>
+        <h1>Admin Page</h1>
+        <h2>Welcome Ben</h2>
+        <h3>About Me</h3>
+    </header>
+    <div class="forms">
+        <div class="container">
+            <h4>Add information</h4><form method="post">
+                <textarea rows="4" cols="100" name="add"></textarea>
+                <input type="submit" value="Add">
+            </form>
+        </div>
+        <div class="container">
+            <h4>Edit information</h4>
+            <p>Select Paragraph to Edit</p>
+                <select>
```

Comment:
Did you mean for this selector to be part of the form below? It's outside it at the moment.